control/bgp.yaml

```diff
@@ -116,4 +121,127 @@ components:
           format: uint32
           maximum: 3600
           default: 30
-          x-field-uid: 2
\ No newline at end of file
+          x-field-uid: 2
+
+    Action.Protocol.Bgp.ImportRoutes:
+      description: |-
+        Import V4 and V6 routes installed on and exported from a DUT,
+        the format for which could be different across vendors, and convert
+        those to OTG V4 or V6 routes configured across corresponding BGP peers.
+        Following additional details need to be considered:
```

Comment:
minor nit, will need to update this description once different formats/conversions have known implementations instead of listing requirements.